### PR TITLE
 contributor count

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###
**1. top part**
image

**2. contributor part**
image

### Checklist ###
  - [x] doc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,9 @@ Participating discussions via the following IM groups is also welcomed.
 Over the past few years, NNI has received thousands of feedbacks on GitHub issues, and pull requests from hundreds of contributors.
 We appreciate all contributions from community to make NNI thrive.
 
+<!-- contributors-NUMBER-orange NUMBER should be rewrite by hand -->
```
Comment (matluster) [inline README.md:92]: You should add this on the top of the page, in the badge list?

Comment (Lijiaoa) [reply README.md:94]: +1

// https://img.shields.io/github/contributors-anon/microsoft/nni

File: README.md [hunk 2]
```diff
@@ -10,6 +10,8 @@
 [![Pull Requests](https://img.shields.io/github/issues-pr-raw/Microsoft/nni.svg)](https://github.com/Microsoft/nni/pulls?q=is%3Apr+is%3Aopen)
 [![Version](https://img.shields.io/github/release/Microsoft/nni.svg)](https://github.com/Microsoft/nni/releases)
 [![Documentation Status](https://readthedocs.org/projects/nni/badge/?version=stable)](https://nni.readthedocs.io/en/stable/?badge=stable)
+[![](https://img.shields.io/github/contributors-anon/microsoft/nni)](https://nni.readthedocs.io/en/stable/?badge=stable)
```
Comment (matluster) [inline README.md:13]: Don't need a link here.

Comment (Lijiaoa) [reply README.md:13]: ok

**Review by ultmaster — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Lijiaoa) [reply README.md]: update link to `github contributors page`

**Review by J-shang — APPROVED** [review]

**Pull request merged**